### PR TITLE
Fixes ghetto chutes

### LINKED ISSUE #4662: Lockers can double as emergency parachutes.
Alt title: "Ghetto Drop-pods."

#### Brief description of the issue
If you are in a locker that is pushed out the back of the shuttle while it is in transit, you land on a planet without any noticeable damage.

#### What you expected to happen
Itty-bitty monkey chunks due to high-speed impact with the planet's surface

#### What actually happened
Not that.

#### Steps to reproduce
1) Get locker.
2) Close self in locker.
3) Have someone push locker out back of ship that is in transit.
4) Fall.
5) Land.

#### Additional info:
 - Server revision: master - 2018-01-26
4b6e64111e6521b3d8e9feb3bbac855aac4cc56f

### PR DESCRIPTION
**Pull request by Anewbe**
Falling from the shuttle in a locker or mech will no longer save you from the fall damage.
Fixes #4662

### CODE REVIEW

**Comment by Techhead0** [comment]
"Ghetto chutes"?
I think you mean "ghetto **_drop pods_**".

**Comment by Anewbe** [comment]
Close enough.

**Comment by mistyLuminescence** [comment]
can we make mechs dropping from low orbit a Feature at some point

**Comment by Anewbe** [comment]
Out of Scope.

File: code/modules/multiz/movement.dm [hunk 1]
```diff
@@ -376,60 +379,32 @@
...
 
-		if(planetary)	//Since the planetary fall damage is calibrated for humans, we need to up this a bit
-			damage_min *= 2
-			damage_max *= 2
-
-		adjustBruteLoss(rand(damage_min, damage_max))
+		// Because wounds heal rather quickly, 10 (the default for this proc) should be enough to discourage jumping off but not be enough to ruin you, at least for the first time.
+		// Hits 10 times, because apparently targeting individual limbs lets certain species survive the fall from atmosphere
+		for(var/i = 1 to 10)
+			adjustBruteLoss(rand(damage_min, damage_max))
```
Comment (Neerti) [inline code/modules/multiz/movement.dm]: Does this increase the damage of regular falls (IE from deck to deck) by ten times?

Comment (Anewbe) [reply code/modules/multiz/movement.dm]: Should actually be slightly weaker.
Old proc added that amount of damage to every limb on the body.

**Pull request merged**